apps/mobile/metro.config.js

```diff
@@ -17,6 +17,6 @@ config.resolver.nodeModulesPaths = [
   path.resolve(workspaceRoot, "node_modules"),
 ];
 // 3. Force Metro to resolve (sub)dependencies only from the `nodeModulesPaths`
-// config.resolver.disableHierarchicalLookup = false;
+config.resolver.disableHierarchicalLookup = true;
```

Comment:
This is how this was intended to be and finally works now that we have nohoist setup properly